### PR TITLE
6.7.0: Backend entities

### LINKED ISSUE #3666: Backend entity created at backend startup
### Description
As of Sensu Go 6.7.0, when customers start the backend, Sensu will automatically create a backend entity, along with the sensu-system namespace.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [x] new doc (requires a completely new doc)
- [ ] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-entities/entities/
https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/backend/
https://docs.sensu.io/sensu-go/latest/operations/control-access/namespaces/

### Additional context
Issues: https://github.com/sensu/sensu-go/issues/4261 and https://github.com/sensu/sensu-go/issues/4443
PR: https://github.com/sensu/sensu-go/pull/4624

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Documents backend entities and the sensu-system namespace.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/3666

Signed-off-by: Hillary Fraley <hfraley@sumologic.com>

### CODE REVIEW

File: content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md [hunk 1]
```diff
@@ -356,6 +356,156 @@ spec:
...
+If a backend and an agent try to create the same entity, the entity class will default to `backend`.
+The information provided by the backend takes precedence over the information provided by the agent.
+The backend should update backend entities to use information from the backend instead of from the agent.
+
+### Backend entity keepalives
+
+Backend entities generate a keepalive event every ...
+
+**TODO**: Need more information about keepalives from backend entities. Also, this question from the original issue: "Question: What should the behaviour be when there are both a backend and an agent generating keep-alive events?"
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md]: I couldn't find answers to the questions in https://github.com/sensu/sensu-go/issues/4443#issuecomment-1020384973 so I added placeholders here and hopefully you can answer in review

File: content/sensu-go/6.7/operations/control-access/namespaces.md [hunk 2]
```diff
@@ -64,6 +64,10 @@ To organize resources, use labels rather than multiple namespaces.
 Every [Sensu backend][1] includes a `default` namespace.
 All resources created without a specified namespace are created within the `default` namespace.
 
+At startup, Sensu also creates the `sensu-system` namespace to contain [backend entities][12].
+The `sensu-system` namespace and backend entities facilitate backend error reporting and make Sensu deployments more observable by enabling backend-generated status and metrics events.
+Only cluster admins have access to the `sensu-system` namespace.
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.7/operations/control-access/namespaces.md:72]: I should probably add something like `If you have cluster admin permissions, you can use sensuctl and the web UI to access backend entities like other entities.` here too

File: content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md [hunk 3]
```diff
@@ -356,6 +356,156 @@ spec:
...
+When a backend [starts up][41], Sensu automatically creates the [`sensu-system` namespace][44] and a new entity with `entity_class: backend`.
+Sensu populates the backend entity with the backend's system information.
+
+The backend uses its own entity to report cluster state errors.
+Backend entities can generate events due to error conditions like unavailable secrets providers or secrets.
+Events generated by a sensu-agent running on the backend host are also associated with the backend entity.
+
+To prevent overloading the event bus with backend events, Sensu generates backend events no more than every 30 seconds, unless the status changes.
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md:370]: I also saw this in the updated code:

```
BackendCheckName is the name fo the check that is created when a backend
	// event is generated from an error.
	BackendCheckName = "backend"
```

Can you tell me more about this check? I'm trying to figure out how/if to document it.

Comment (fguimond) [reply content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md:370]: Good catch Hilary! This is actually not used anymore and will need to be removed. The idea was that events created against the backend entity would use "backend" as check name, however we ended up going more granular and use something more generic. Currently events are created for Vault access errors and we use "secrets" as the check name.

File: content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md (hunk 1 above)
Comment (fguimond) [reply content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md]: No keepalive events are generated for backend entities.

File: content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md [hunk 4]
```diff
@@ -356,6 +356,156 @@ spec:
...
+### Backend entity keepalives
+
+Backend entities generate a keepalive event every ...
+
+**TODO**: Need more information about keepalives from backend entities. Also, this question from the original issue: "Question: What should the behaviour be when there are both a backend and an agent generating keep-alive events?"
+
+### Delete a backend entity
+
+**TODO**: Need more information about how to delete a backend entity. It's not clear whether either of the two ways described in the original issue (see below) were implemented.
```
Comment (fguimond) [inline content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md]: Backend entities can be deleted manually by admin users.

File: content/sensu-go/6.7/operations/control-access/namespaces.md (hunk 2 above)
Comment (fguimond) [reply content/sensu-go/6.7/operations/control-access/namespaces.md:72]: Correct.

**Pull request merged**